common/meta/raft-store/src/config.rs

```diff
@@ -29,109 +29,68 @@ pub const KVSRV_BOOT: &str = "KVSRV_BOOT";
 pub const KVSRV_SINGLE: &str = "KVSRV_SINGLE";
 pub const KVSRV_ID: &str = "KVSRV_ID";
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, StructOpt, StructOptToml)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Parser)]
 pub struct RaftConfig {
-    /// Identify a config. This is only meant to make debugging easier with more than one Config involved.
-    #[structopt(long, default_value = "")]
+    /// Identify a config.
+    /// This is only meant to make debugging easier with more than one Config involved.
+    #[clap(long, default_value = "")]
     pub config_id: String,
 
-    #[structopt(
-        long,
-        env = KVSRV_API_HOST,
-        default_value = "127.0.0.1",
-        help = "The listening host for metadata communication"
-    )]
+    /// The listening host for metadata communication.
+    #[clap(long, env = KVSRV_API_HOST, default_value = "127.0.0.1")]
     pub raft_api_host: String,
 
-    #[structopt(
-        long,
-        env = KVSRV_API_PORT,
-        default_value = "28004",
-        help = "The listening port for metadata communication"
```

Comment:
Why remove help here?

Reply:
The `help` will be helpful with:
databend-meta -h

Reply:
the doc will be help info